### ISSUE TITLE
Undefined variable $params

### ISSUE THREAD

**Issue by mosdraev** [cakemail/cakemail-php#1]
These issues are on `Cakemail\Subapis\UserApi::getSelf()` and `Cakemail\Subapis\AccountApi::getSelf()` 

When `getSelf()` is called using $api->account->getSelf(); it returns Undefined variable $params

Sample code from UserApi class

```
public function getSelf()
{
        if (gettype($params) != 'array' && gettype($params) != 'NULL') {
            throw new ApiException('Parameter must be an array');
        }

        return new Response($this->openApiObj->getSelfUser());
}
```

`$params` variable has no point of reference.

**Comment by vlamanna** [comment]
Thanks for reporting, we will work on a fix for this.

**Comment by demetrius-edelin** [comment]
Hello,

The bug is fixed now in 1.2.1

Best regards.
